mscore/importgtp-gp4.cpp

```diff
@@ -60,12 +60,12 @@ namespace Ms {
 bool GuitarPro4::readMixChange(Measure* measure)
       {
       /*char patch   =*/ readChar();
-      char volume  = readChar();
-      char pan     = readChar();
-      char chorus  = readChar();
-      char reverb  = readChar();
-      char phase   = readChar();
-      char tremolo = readChar();
+      signed char volume  = readChar();
+      signed char pan     = readChar();
+      signed char chorus  = readChar();
+      signed char reverb  = readChar();
+      signed char phase   = readChar();
+      signed char tremolo = readChar();
```

Comment:
the subsequent if statements always evaluate to true when these chars are unspecified on arm.